astroquery/gemini/core.py

```diff
@@ -106,6 +111,28 @@ def __init__(self, *args):
         """
         super().__init__()
 
+    def _login(self, username, password):
+        """
+        Login to the Gemini Archive website.
+
+        This method will authenticate the session as a particular user.  This may give you access
+        to additional information or access based on your credentials
+
+        Parameters
+        ----------
+        username : str
+            The username to login as
+        password : str
+            The password for the given account
+        """
+        params = dict(username=username, password=password)
+        r = self._session.request('POST', 'https://archive.gemini.edu/login/', params=params)
+        if b'<P>Welcome, you are sucessfully logged in' not in r.content:
```

Comment:
I think this can be an error instead of a warning, after all the user needed to ask for authentication. I see that MAST does the same, but nevertheless I think we should rather change that, too.

Reply:
ok done